editor/components/index.js

```diff
@@ -5,6 +5,7 @@ export { default as AlignmentToolbar } from './alignment-toolbar';
 export { default as BlockAlignmentToolbar } from './block-alignment-toolbar';
 export { default as BlockControls } from './block-controls';
 export { default as BlockEdit } from './block-edit';
+export { withBlockEditContext } from './block-edit/context';
```

Comment:
I'm still on the fence here, what's the difference between making this available to block authors and passing its content as props to `edit` like we do for several other props :)

Reply:
cc @aduth Thoughts?

Reply:
We can do `prop drilling` for blocks, too :)
I wasn't happy about exposing this HOC to be honest.